### ISSUE TITLE
extract PCI standard to its own repository

### ISSUE THREAD

**Issue by afeld** [opencontrol/compliance-masonry#209]
There are a handful of PCI files in this repository that seem like they could be useful as a standalone [standard](https://github.com/opencontrol/schemas#standards) repository, akin to https://github.com/opencontrol/NIST-800-53-Standards. No strong feelings about whether they also stay in this repository as fixtures.

/cc @Gergues https://github.com/opencontrol/schemas/issues/46

**Comment by geramirez** [comment]
@afeld Just for some context, the fixtures were added to make sure that masonry works with standards which contain control names that could be interpreted as a floats on import. ie (1.1). Since Go is statically typed this shouldn't be an issue. 

Regardless, I've made a [PCI certifications repo](https://github.com/geramirez/PCI-DSS-Certifications) if it looks good to @Gergues and ya'll I can add it to opencontrol.
